client/packages/programs/src/RnRForms/ListView/AppBarButtons.tsx

```diff
@@ -20,7 +20,6 @@ export const AppBarButtons = ({
     <AppBarButtonsPortal>
       <Grid container gap={1}>
         <ButtonWithIcon
-          startIcon={<PlusCircleIcon />}
```

Comment:
side quest: on small screens, having both `startIcon` and `Icon` meant the plus was showing twice?